be/docker/be-dev.dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:lts-alpine
+
+WORKDIR /app
+
+# append node_modules to PATH
+ENV PATH /app/node_modules/.bin:$PATH
```

Comment:
You'll also need to add the variables from (gitignored) `.env` to the container,

Reply:
Fixed in new commit: 8ffdf9d8def4e27d69b9dfaeaf6d21dcdc414892.